meta/acronyms.txt

```diff
@@ -89,3 +89,5 @@ WWW - World Wide Web
 XG - (*,G) Any-source Multicast
 XOFF - Transmitter Off
 XON - Transmitter On
+NRZ - Non Return to Zero
+PAM4 - Pulse Amplitude Modulation
```

Comment:
I think we can just keep PAM here, PAM4 is more specific case.

Reply:
Updated the acronym. Added PAM4 as well to make the build meta check successful.